spec/index.html

```diff
@@ -6238,16 +6238,203 @@ <h5>LANG</h5>
                 </div>
               </div>
             </div>
+
+            <p>
+              Function examples:
+            </p>
+            <div class="result">
+              <table>
+                <thead>
+                  <tr>
+                    <th>Expression</th>
+                    <th>Result</th>
+                  </tr>
+                </thead>
+                <tbody>
+                  <tr>
+                    <td><code>LANG("abc"@en)</code></td>
+                    <td><code>"en"</code></td>
+                  </tr>
+                  <tr>
+                    <td><code>LANG("abc"@en--ltr)</code></td>
+                    <td><code>"en"</code></td>
+                  </tr>
+                  <tr>
+                    <td><code>LANG("abc")</code></td>
+                    <td><code>""</code></td>
+                  </tr>
+                  <tr>
+                    <td><code>LANG(1)</code></td>
+                    <td><code>""</code></td>
+                  </tr>
+                  <tr>
+                    <td><code>LANG(&lt;http://example/&gt;)</code></td>
+                    <td><code>error</code></td>
+                  </tr>
+                </tbody>
+              </table>
+            </div>
+          </section>
+
+          <section id="func-langdir">
+            <h5>LANGDIR</h5>
+            <pre class="prototype nohighlight"> <span class="return">xsd:string</span>  <span class="operator">LANGDIR</span> (<span class="type"><span class="type literal">literal</span></span> <span class="name">ltrl</span>)
+            </pre>
+            <p>
+              If the argument is a literal,
+              the function <code>LANGDIR</code> returns the
+              <a data-cite="RDF12-CONCEPTS#dfn-base-direction">base direction</a>
+              of the literal with datatype <code>rdf:dirLangString</code>.
+              Otherwise, the function returns the empty string.
```

Comment:
First part done.

"If - Otherwise," is used in many other places and I'd like to keep that style.
"For all other"  is not an appropriate wording in every other place.